integrations/vello_svg/src/geom.rs

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.

```

Comment:
The SPDX header type is explicitly mentioned as allowed in the [MPL FAQ under Q27](https://www.mozilla.org/en-US/MPL/2.0/FAQ/#machine-readable-header) and also [under Q4](https://www.mozilla.org/en-US/MPL/2.0/FAQ/#apply). Also [MPL section 3.4](https://www.mozilla.org/en-US/MPL/2.0/#notices) states that *You may not remove or alter the **substance** of any license notices*.

Thus the MPL header can be replaced with a header that matches our style while keeping the same substance, as the SPDX format is explicitly allowed and conveys the same meaning.





```suggestion
// Copyright 2018 Yevhenii Reizner
// SPDX-License-Identifier: MPL-2.0
```